op-service/enum/enum.go

```diff
@@ -1,36 +1,18 @@
 package enum
 
 import (
-	"fmt"
 	"strings"
 )
 
-// Stringered wraps the string type to implement the fmt.Stringer interface.
-type Stringered string
-
-// String returns the string value.
-func (s Stringered) String() string {
-	return string(s)
-}
-
-// StringeredList converts a list of strings to a list of Stringered.
-func StringeredList(values []string) []Stringered {
-	var out []Stringered
-	for _, v := range values {
-		out = append(out, Stringered(v))
-	}
-	return out
-}
-
 // EnumString returns a comma-separated string of the enum values.
 // This is primarily used to generate a cli flag.
-func EnumString[T fmt.Stringer](values []T) string {
+func EnumString[T ~string](values []T) string {
 	var out strings.Builder
 	for i, v := range values {
-		out.WriteString(v.String())
+		out.WriteString(string(v))
 		if i+1 < len(values) {
 			out.WriteString(", ")
 		}
 	}
 	return out.String()
-}
+}
```

Comment:
The linter seems to want this line ending back.  If you run:
```
gofmt -w op-service/enum/enum.go
```
it should fix up the formatting for you.